Android/PushTutorial/settings.gradle

```diff
@@ -1 +1 @@
-include ':app'
+include ':push-tutorial-one'
```

Comment:
this renaming shouldn't be here. Unfortunately we need to have it in the correct namespace from the start.